github_oidc_provider.tf

```diff
@@ -0,0 +1,9 @@
+module "github_oidc_provider" {
+  count  = local.github_oidc_enabled ? 1 : 0
+  source = "./modules/github_oidc_provider"
+
+  s3_access_logging_bucket_names = [
+    module.eu-west-1.access_logging_bucket.bucket,
+    module.eu-west-2.access_logging_bucket.bucket
+  ]
```

Comment:
we should specify which provider that passed into the accounts module should be used here. 

I think this would require a configuration alias in the module's versions.tf, and therefore assign a provider to every resource and data source.